app/templates/views/edit-user-permissions.html

```diff
@@ -14,7 +14,7 @@ <h1 class="heading-large">
   </h1>
 
   <p>
-    {{ user.email_address }}
+    {{ user.email_address }} <a href="{{ url_for('.edit_user_email', service_id=current_service.id, user_id=user.id)}}">Change</a>
```

Comment:
Putting a little bit more space between the email address and the link will look nicer (an em-space is a space the width of the letter ‘M’):
```suggestion
    {{ user.email_address }}&emsp;<a href="{{ url_for('.edit_user_email', service_id=current_service.id, user_id=user.id)}}">Change</a>
```

Reply:
cool 👍 . Like this, aye?

<img width="994" alt="screen shot 2019-02-21 at 10 43 17" src="https://user-images.githubusercontent.com/20957548/53163242-8f44f000-35c5-11e9-8c18-1c02f3ba1fc6.png">